Gds/src/fprime_gds/flask/static/js/vue-support/uplink.js

```diff
@@ -94,7 +94,7 @@ Vue.component("uplink", {
     },
     computed: {
         /**
-         * Returns the total list of files for display in the table. In short, this is curatec + uplinked (reversed)
+         * Returns the total list of files for display in the table. In short, this is curated + uplinked (reversed)
```

Comment:
I think this is right, I spent a while scratching my head about it.

Reply:
I think so!